bugalink-backend/users/views.py

```diff
@@ -74,7 +74,7 @@ def delete(self, request, *args, **kwargs):
         user.email = new_email
         user.first_name = "Usuario eliminado"
         user.last_name = ""
-        user.photo = ""
+        user.photo = user.photo.delete()
```

Comment:
You are deleting the file and assigning it to a variable? Remove the `user.photo = ` part and only keep the code for deleting